test/day11Arrays2d.js

```diff
@@ -0,0 +1,28 @@
+const getArray = input => {
+  const arr = []
+  for (let i = 0; i < 6; i++) {
```

Comment:
no deberia ser input.lenght?
y en lugar de un for hacer
input.map(value =>  value.split('')).map(Number)

algo asi?

Reply:
Si de hecho queda mucho mas bonito esa parte no la habia tocado porque hacia venia puesta en hackerRank pero ya refactorie a map